app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
```

Comment:
1. Xóa code ` = ""`
2. `empty?` dùng `blank?` thay thế nhé
Phân biệt cho chị blank? empty? và nil? có gì khác nhau nhé

3. line  4 -> 8: if - else
em dùng toán tử ngôi số 3 nhé: `condition ? result_for_true : result_for_false`